demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -780,7 +780,7 @@ def format_user_input(self) -> Dict[str, str]:
             if line.startswith(entity_identifier):
                 entity_name = line.lstrip(entity_identifier)
                 if items_path.get(entity_name):
-                    entity_name = get_display_name(items_path.get(entity_name))
+                    entity_name = get_display_name(items_path[entity_name])
```

Comment:
there is always `entity_name` key in the `items_path` dict?

Reply:
The condition is:
```
if items_path.get(entity_name):
    entity_name = get_display_name(items_path[entity_name])
```
So this line is executed only if `items_path.get(entity_name)` equals true.